src/pharmpy/modeling/error.py

```diff
@@ -597,3 +597,47 @@ def set_dtbs_error_model(model, fix_to_log=False):
     model.observation_transformation = obs
 
     return model
+
+
+def add_time_varying_error_model(model, cutoff, idv='TIME'):
```

Comment:
Perhaps we should rename it to `set_time_varying_error_model` as we actually change the model to become time varying.